priv/templates/nodetool

```diff
@@ -86,6 +86,22 @@ main(Args) ->
                     io:format("RPC to ~p failed: ~p~n", [TargetNode, Reason]),
                     halt(1)
             end;
+        ["ping"] ->
```

Comment:
Why did you add these to nodetool?

Reply:
_I have also fixed "nodetool ping ..etc.." which was performing the ping correctly, but then outputting:_

```
Other: ["ping"]
Usage: nodetool {ping|stop|restart|reboot|rpc|rpcterms|eval [Terms]} [RPC
```

Reply:
That shouldn't be the case, something else is wrong if you are seeing the usage. The extended start script uses `nodetool rpc` to do all of the commands, so usage should not be printed because commands like `nodetool ping` should never be called.

Reply:
The current **extended_bin_windows** is calling nodetool ping:
https://github.com/erlware/relx/blob/4.0.0/priv/templates/extended_bin_windows#L323

I can update these to use rpc instead if you would like?
But.. surely "nodetool ping" should be useable too?

Reply:
Ooh, woops. I thought I did updates to the windows script but I must have missed some stuff. You can see in the `extended_bin` script that those commands have been replaced by just `rpc` because on OTP-23+ we'll be using `erl_call` instead of `nodetool` and eventually nodetool will be removed https://github.com/erlware/relx/blob/4.0.0/priv/templates/extended_bin#L240

Reply:
Ok, no more nodetool ping, only rpc - added eval support too